.github/config.yml

```diff
@@ -12,6 +12,7 @@ newIssueWelcomeComment: |
 newPRWelcomeComment: |
  Thanks for opening this pull request!
  There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
```

Comment:
Oh wait, this line already says "We're here to help". Can it say "is-reviewers will be here to help!"?

Reply:
ok